### DISCUSSION TITLE
Using Polls instead of issues to vote for new icons?

### DISCUSSION THREAD

**Discussion by GwynethLlewelyn** [FortAwesome/Font-Awesome#19302] [General]
GitHub seems to have this nice new feature, 🗳 Polls. I wonder if the clutter on the Issues, where votes of icons are recorded wrongly so often (because people don't bother to read the instructions), mixed in with 'real' reports of issues, as well as feature requests not related to new icons, couldn't be best served by this nifty new GitHub feature?

What do you think?

(**Note:** there would remain the issue of moving all existing icon requests to 🗳 Polls... I don't know if that's possible to do automatically!)

**Comment by sensibleworld** [comment]
I like this idea, @GwynethLlewelyn. The current system is less than ideal, and makes it hard to track current requests/needs. I'm going to do some research on potential improvements.

**Comment by sensibleworld** [comment]
A MUCH belated follow-up to say that we've just moved new icon requests to discussions, and will be working to port the older icon requests in the near-ish future. This should help declutter our issues.
